src/lang/pl-pl.json

```diff
@@ -367,10 +367,10 @@
 	"remove": "Usuń",
 	"quicktools:command-palette": "Paleta poleceń",
 	"default file encoding": "Domyślne kodowanie plików",
-	"remove entry": "Czy na pewno chcesz usunąć '{nazwa}' z zapisanych ścieżek? Należy pamiętać, że usunięcie go nie spowoduje usunięcia samej ścieżki.",
-	"delete entry": "Potwierdź usunięcie: '{nazwa}'. Tej akcji nie można cofnąć. Kontynuować?",
+	"remove entry": "Czy na pewno chcesz usunąć '{name}' z zapisanych ścieżek? Należy pamiętać, że usunięcie go nie spowoduje usunięcia samej ścieżki.",
+	"delete entry": "Potwierdź usunięcie: '{name}'. Tej akcji nie można cofnąć. Kontynuować?",
```

Comment:
Why english words?

Reply:
I thought it was a variable that could not be translated. If it can be translated then I will correct it soon.

Reply:
corrected

Reply:
Yeah , sorry those are variables (I was confused)

Reply:
No need to correct it

Reply:
ok, I restored the original phrases for the variables, they are in English now. You can merge it.